File: documentapi/src/vespa/documentapi/messagebus/messages/documentignoredreply.cpp
// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
#include "documentignoredreply.h"
#include <vespa/documentapi/messagebus/documentprotocol.h>

namespace documentapi {

DocumentIgnoredReply::DocumentIgnoredReply()
    : DocumentReply(DocumentProtocol::REPLY_DOCUMENTIGNORED)
{
}

}

File: documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentIgnoredReply.java
// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
package com.yahoo.documentapi.messagebus.protocol;

public class DocumentIgnoredReply extends DocumentReply {
    public DocumentIgnoredReply() {
        super(DocumentProtocol.REPLY_DOCUMENTIGNORED);
    }
}
